server/services/interventionsService.ts

```diff
@@ -516,4 +522,16 @@ export default class InterventionsService {
     })) as Appointment
   }
  */
+
+  async getServiceProviderReportingData(
+    token: string,
+    reportDates: CreateReportDateParams
+  ): Promise<ServiceProviderReportReferral[]> {
+    const restClient = this.createRestClient(token)
+
+    return (await restClient.get({
+      path: `/performance-report?from=${reportDates.fromDate}&to=${reportDates.toDate}`,
```

Comment:
`get` takes a query argument which is a bit neater

```
query: { from: ..., to: ... }
```

Reply:
Ah yeah, I initially had this as a `post` which didn't but forgot to update this after switching to GET

Reply:
Have updated this.